### ISSUE TITLE
Allow role attribute on components

### ISSUE THREAD

**Issue by dsl101** [quasarframework/eslint-plugin-quasar#8]
**Describe the bug**
Not _entirely_ sure if this is a bug, but I need to set the html `role` attribute on a Quasar component, and it throws an invalid prop error.

**To Reproduce**
Add the `role` html attribute to any Quasar component:
```
<q-btn role="link" />
```

**Expected behavior**
That the attribute would be accepted without error.

With single-line components, it's relatively easy to disable the rule for that line (which is not ideal, but manageable):
```
<!-- eslint-disable-next-line quasar/no-invalid-props -->
<q-btn role="link" />
```

But with multiline components, this isn't possible:
![image](https://user-images.githubusercontent.com/900542/148204753-ae5ce818-f6e2-4ecd-9ccc-24e36f5230af.png)

**Comment by rstoenescu** [comment]
You need to disable the ESLint rule. Go to /.eslintrc.js and add: `'quasar/no-invalid-props': 'off'`

**Comment by dsl101** [comment]
So, if I'm using that HTML attribute in one place, I have to disable the entire rule?

Edit: To be clear, I don't want to disable the rule. I want to get rid of this 'false positive'. Seems like the plugin should allow valid HTML attributes even if they're not technically props of the component, no?

**Comment by rstoenescu** [comment]
You are right, but the eslint plugin does not supports that due to the huge maintenance (and initial implementation cost) that it involves (which we currently cannot afford).

**Comment by hawkeye64** [comment]
I'll have a fix for this released shortly. We have a list of DOM attributes we check against. The "role" attribute (among a few others) is missing.

**Comment by dsl101** [comment]
Yeah, I was just looking at `domProps` and `vueProps` and thinking it could be extended with `htmlProps` or something similar. That would be fab.

**Comment by hawkeye64** [comment]
Unfortunately, for some reason, I am unable to transfer this to the `eslint-plugin-quasar` repo...

**Comment by hawkeye64** [comment]
Fixed in v1.1.0
